text/0000-exhaustive-integer-pattern-matching.md

```diff
@@ -0,0 +1,89 @@
+- Feature Name: `exhaustive_integer_patterns`
+- Start Date: 10/11/2018
+- RFC PR:
+- Rust Issue: [#50907](https://github.com/rust-lang/rust/issues/50907)
+
+# Summary
+[summary]: #summary
+
+Extend Rust's pattern matching exhaustiveness checks to cover the integer types: `u8`, `u16`, `u32`, `u64`, `u128`, `usize`, `i8`, `i16`, `i32`, `i64`, `i128`, `isize` and `char`.
```

Comment:
It's not obvious what is the range for `char` - full `u32`, continuous range permitted by Unicode, range permitted by Unicode with holes for surrogates/something else?